### DISCUSSION TITLE
how to bring stroke back or top?

### DISCUSSION THREAD

**Discussion by MiltonBalaOfficial** [xournalpp/xournalpp#5924] [Q&A] [answered]
how to do that?
![image](https://github.com/user-attachments/assets/6a26c1c9-e743-43c9-acc5-e338aa2accf9)

**Comment by rolandlo** [comment, marked as answer]
Select the blue stroke and use menu `Edit > Arrange Selection > Send to Back`.

**Reply by MiltonBalaOfficial** [reply]
Hi, I am a user not a developer, but I feel the need of an icon for the "Bring forward" or "Send Backward". Somehow I managed to add an icon through a plugin but unable to link the button with the action. May I get some help?

**Reply by rolandlo** [reply]
`app.uiAction({action = "ACTION_ARRANGE_SEND_TO_BACK"})` should do the job in the callback.

**Reply by MiltonBalaOfficial** [reply]
Thanks, It is working nicely. But where to get the name of other actions?

**Reply by rolandlo** [reply]
In `Control::actionPerformed` as documented on the [website](https://xournalpp.github.io/guide/plugins/plugins/#using-plugins-to-define-shortcuts-for-ui-actions)
